### ISSUE TITLE
Pytorch 0.4.0 documentation typo for batchnorm

### ISSUE THREAD

**Issue by bado-lee** [pytorch/pytorch#5702]
This issue is related to
https://github.com/pytorch/pytorch/issues/5625

![image](https://user-images.githubusercontent.com/26222919/37268701-8d1baae2-260a-11e8-99d7-4087df60e64c.png)

Equation is fixed but X(Multiplication Sign) is lost in the fixed documentation which makes harder to read compared to previous one.

![image](https://user-images.githubusercontent.com/26222919/37268726-b3a1ddd0-260a-11e8-8691-914bf4ce85d9.png)

![image](https://user-images.githubusercontent.com/26222919/37268802-16c16dd6-260b-11e8-9ea2-247c78e7bab5.png)
This is expected instead of
![image](https://user-images.githubusercontent.com/26222919/37268811-1e86b9ae-260b-11e8-810c-b4baf35962d7.png)

**Comment by vishwakftw** [comment]
This will be fixed in https://github.com/pytorch/pytorch/pull/5685. Thanks for bringing it up.

**Comment by bado-lee** [comment]
Thanks for the info.

**Comment by SsnL** [comment]
@soumith this can be closed
